src/components/cc-orga-member-list/cc-orga-member-list.js

```diff
@@ -21,6 +21,8 @@ import { linkStyles } from '../../templates/cc-link/cc-link.js';
  * @typedef {import('./cc-orga-member-list.types.js').OrgaMemberInviteFormState} OrgaMemberInviteFormState
  * @typedef {import('./cc-orga-member-list.types.js').OrgaMemberListState} OrgaMemberListState
  * @typedef {import('./cc-orga-member-list.types.js').Authorisations} Authorisations
+ * @typedef {import('./cc-orga-member-list.types.js').InviteMember} InviteMember
+ * @typedef {import('./cc-orga-member-list.types.js').OrgaMemberCardState} OrgaMemberCardState
```

Comment:
**fix:** `cc-orga-member-list.types.d.ts` does not export this :thinking:
We should either:
1. move it to `common`,
2. import it from `cc-orga-member-card.types.d.ts`,
3. export it and import it from `cc-orga-member-list.types.d.ts` `export type OrgaMemberCardState = OrgaMemberCardState;`.